x-pack/plugins/security_solution/public/common/containers/cases/use_get_related_cases_by_event.ts

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { useCallback, useState, useEffect } from 'react';
+import { useKibana, useToasts } from '../../lib/kibana';
+import { CASES_ERROR_TOAST } from '../../components/event_details/insights/translations';
+import { APP_ID } from '../../../../common/constants';
+
+type RelatedCases = Array<{ id: string; title: string }>;
+
+export const useGetRelatedCasesByEvent = (eventId: string) => {
```

Comment:
This can be re-used in the alert details flyout, but didn't want to increase the size of this pr